src/OrchardCore.Modules/OrchardCore.ContentFields/Views/TextField-Color.Edit.cshtml

```diff
@@ -4,14 +4,64 @@
     var settings = Model.PartFieldDefinition.GetSettings<TextFieldSettings>();
 }
 
-<div class="@Orchard.GetFieldWrapperClasses(Model.PartFieldDefinition)" id="@Html.IdFor(x => x.Text)_FieldWrapper">
-    <label asp-for="Text" class="@Orchard.GetLabelClasses(inputRequired: settings.Required)">@Model.PartFieldDefinition.DisplayName()</label>
-    <div class="@Orchard.GetEndClasses("pt-lg-2")">
-        <input asp-for="Text" class="form-control content-preview-text" type="color" style="min-height: 2.25rem" />
-        <span asp-validation-for="Text"></span>
-        @if (!string.IsNullOrEmpty(settings.Hint))
-        {
-            <span class="hint">@settings.Hint</span>
+<style at="Head">
```

Comment:
The only thing I don't like is these inline styles and scripts. We should try to put these into files moving forward so that it makes management/refactoring easier. We eventually want these to be added as ES6 components.

Reply:
There are many places that we're using such styles and scripts, but we could refactor them in a separated PR